app/server.js

```diff
@@ -58,12 +47,17 @@ ttn
     //on uplink messages
     client.on("uplink", function(devID, payload) {
       //print the message in the console
-      if (payload.payload_fields.bssid3 && setAddresses == false) {
+      if (setAddresses == false) {
         console.log("Received uplink from ", devID);
         console.log(payload.payload_fields);
-        macAddress1 = payload.payload_fields.bssid1;
-        macAddress2 = payload.payload_fields.bssid2;
-        macAddress3 = payload.payload_fields.bssid3;
+        //iterate over each bssid discovered
+        for (var field in payload.payload_fields) {
+          address = payload.payload_fields[field];
+          //appends the bssid tab
+          wiFiBssids.push({
```

Comment:
If you never clear/reset the array, you'll keep pushing and pushing to an ever growing array